extensions/kogito/deployment/src/main/java/io/quarkus/kogito/deployment/KogitoAssetsProcessor.java

```diff
@@ -89,7 +90,8 @@ private boolean hotReload(LaunchMode launchMode) {
     }
 
     private void compileAndRegister(ArchiveRootBuildItem root, Collection<GeneratedFile> generatedFiles,
-            BuildProducer<GeneratedBeanBuildItem> generatedBeans, LaunchMode launchMode)
+            BuildProducer<GeneratedBeanBuildItem> generatedBeans, BuildProducer<ReflectiveClassBuildItem> reflectiveClass,
```

Comment:
You added that reflectiveClass argument that seems not to be used anywhere. Am I missing something?

Reply:
this was left over, I removed it completely as it was injected into build step but not used at all